src/web-components/dropdown/index.ts

```diff
@@ -120,9 +132,12 @@ export class Dropdown extends WebComponentsBaseElement {
         </div>
       </div>
       <div class="dropdown-list">
-        <ul class=${classMap(menuClasses)}>
-          ${options}
-        </ul>
+        <div class=${classMap(menuClasses)}>
+          <div class="header">${header()}</div>
+          <ul class="items">
+            ${options}
+          </ul>
+        </div>
```

Comment:
Return all header from function. 

```suggestion
        <div class=${classMap(menuClasses)}>
          ${header()}
          <ul class="items">
            ${options}
          </ul>
        </div>
```